src/main/java/ch/wisv/payments/exception/ProductGroupInUseException.java

```diff
@@ -0,0 +1,7 @@
+package ch.wisv.payments.exception;
+
+public class ProductGroupInUseException extends RuntimeException {
+    public ProductGroupInUseException(String s) {
```

Comment:
You unfortunately have to do this to make use of the `super(...)` constructor though.